app.js

```diff
@@ -20,22 +20,23 @@ const serveFileToHtml = require('./src/fileServer.js');
 const initialize = require('./src/initializer.js');
 const db = require('./src/db-conn.js');
 const dbInteract = require('./src/auth.js');
+const constants = require('./constants/index');
 
 const app = express();
-const PORT = argv.port || 8080;
-const ADDRESS = argv.address || '127.0.0.1';
-const connURL = argv.mongo || "mongodb://127.0.0.1:27017/keys";
+const PORT = argv.port || constants.PORT;
+const ADDRESS = argv.address || constants.ADDRESS;
+const connURL = argv.mongo || constants.URI;
 
-const DAY_IN_MS = 86400000;
+const DAY_IN_MS = constants.DAY_IN_MS;
 // times specified in ms
-const CLEARING_MAX_AGE = DAY_IN_MS * 7; // week
+const CLEARING_MAX_AGE = DAY_IN_MS * constants.WEEK; // week
 const CLEARING_MIN_AGE = DAY_IN_MS; // day
-const CLEARING_FREQUENCY = DAY_IN_MS / 3;
+const CLEARING_FREQUENCY = DAY_IN_MS / constants.FREQUENCY;
 const FILE_DIR = path.join(__dirname, '/files/');
```

Comment:
This has to be part of the constant file too.

Reply:
Which variable?

Reply:
FILE_DIR

Reply:
Actually the commit hook which you set was giving the error as then DAY_IN_MS variable was not used.

Reply:
Okey, I will look at it after this is merged then. Thanks.